configure.py

```diff
@@ -805,7 +805,7 @@ def get_nasm_version(asm):
 
 def get_llvm_version(cc):
   return get_version_helper(
-    cc, r"(^(?:FreeBSD )?clang version|based on LLVM) ([0-9]+\.[0-9]+)")
```

Comment:
For detecting the support of OpenSSL's functionality, `configure.py` checks `xcode_version>=5.0` or `llvm_version>=3.3` in https://github.com/nodejs/node/blob/master/configure.py#L1330
Thus, it changes the detection result for ex. `Apple clang version 4.0`.

Thus, if you need to check for `Alpine`, below one would be better.
```diff
- cc, r"(^(?:FreeBSD )?clang version|based on LLVM) ([0-9]+\.[0-9]+)")
+ cc, r"(^(?:(?:FreeBSD|Alpine) )?clang version|based on LLVM) ([0-9]+\.[0-9]+)")
```